src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -6496,6 +6503,56 @@ private BoundExpression BindLeftIdentifierOfPotentialColorColorMemberAccess(Iden
             return boundValue;
         }
 
+        private BoundExpression RebindColorColorConstField(IdentifierNameSyntax left, BoundExpression boundValue, FieldSymbol leftSymbol)
+        {
+            if (!leftSymbol.IsConst)
+            {
+                return boundValue;
+            }
+
+            var memberAccessParent = left.Parent as MemberAccessExpressionSyntax;
+            Debug.Assert(memberAccessParent is not null);
+
+            var targetBoundValue = boundValue;
+
+            var binderSymbol = ContainingType;
+            Debug.Assert(binderSymbol is TypeSymbol);
+
+            var firstOuterScopeBinder = this;
+            while (true)
+            {
+                firstOuterScopeBinder = firstOuterScopeBinder.Next;
+
+                if (firstOuterScopeBinder is null)
+                {
+                    break;
+                }
+
+                // We found the first container that does not apply to our same symbol
+                if (!binderSymbol.Equals(firstOuterScopeBinder.ContainingType))
+                {
+                    break;
+                }
+            }
+
+            // The first in container binder reflects the current scope, so we begin at the outer scope of the binder
+            for (var binder = firstOuterScopeBinder; binder is not null; binder = binder.Next)
+            {
+                // While trying to find a suitable binder for the expression, we do not report diagnostics
+                targetBoundValue = binder.BindMemberAccess(memberAccessParent, false, false, BindingDiagnosticBag.Discarded);
```

Comment:
>false, false [](http://example.com/codeflow?start=79&length=12)

It is not obvious why ```false, false``` are the right values to pass here.

Reply:
Because I think we will never fall into the case of rebinding the LHS of a MAE that is a constant field, if the expression is indexed or invoked. But I'm not entirely sure about that either.